docs/node.md

```diff
@@ -99,25 +99,30 @@ Node Controller is unable to provision the node for you, i.e. it won't install
 any binary; therefore, to
 join Kubernetes cluster, you as an admin need to make sure proper services are
 running in the node. In the future, we plan to automatically provision some node
-services. In case of no cloud provider, Node Controller simply registers all
-machines from `--machines` flag, any further interactions need to be done manually
-by using `kubectl`. If you are paranoid, leave `--machines` empty and create all
-machines from `kubectl` one by one - the two approaches are equivalent.
-Optionally you can skip cluster-wide node synchronization with
-'--sync_nodes=false' and can use REST api/kubectl cli to add/remove nodes.
-
-Node life-cycle management in the Node Controller is still under development, it
-is supposed to manage the Node Status Specification defined above.
-
-### Manual Node Administration
-
-A Kubernetes administrator typically uses `kubectl` to manage `Node`. Similar
-to Node Controller, `kubectl` command only creates/deletes node representation.
-Note if Kubernetes is running on cloud provider, `kubectl create` a node will
-be refused if Node Controller has already synchronized nodes from cloud provider.
-Admin can choose to make the node unschedulable using `kubectl`. Unscheduling the node
-will not affect any existing pods on the node but it will disable creation of
-any new pods on the node. Node unschedulable example:
+services. 
+
+### Registration of nodes
```

Comment:
We should mention the default behavior. Will nodes auto register themselves if the `--register-node` flag is not set to `0`?